### DISCUSSION TITLE
How to make an Advanced Assignment.

### DISCUSSION THREAD

**Discussion by LAPIII** [AntiMicroX/antimicrox#543] [Q&A] [answered]
I've searched the wiki and cannot find out how to do this.

**Comment by pktiuk** [comment, marked as answer]
I hope this image will be self-explanatory   
![image](https://user-images.githubusercontent.com/45544416/194168340-f95c525a-03d4-4d92-9547-2131f1470c85.png)

**Reply by LAPIII** [reply]
Thank you, I don't know how I didn't see that. I'm trying to make the left trigger send `Alt + Tab`. It's not working.

Edit, also I want `Alt + Tab` held as in Windows so that applications can be cycled.

**Reply by pktiuk** [reply]
Idk how this shortcut is supposed to be pressed, maybe Alt Tab shouldn't be clicked at once, but firstly alt and after a while tab (while still holding tab)    
Maybe this will come in handy https://github.com/AntiMicroX/antimicrox/wiki/Advanced-Slot-Explanation

**Comment by LAPIII** [comment]
How did you set dark mode?

**Reply by pktiuk** [reply]
It depends on your operating system. In the case of Linux, it is a standard feature in most of the distros. Most of the apps try to align to a theme that is used system-wide.    
All my apps look like that:
![image](https://user-images.githubusercontent.com/45544416/197018944-2e247679-cc47-4009-a90d-45c247189815.png)
It doesn't seem to be the case for Windows, so there is a dedicated issue for that 
https://github.com/AntiMicroX/antimicrox/issues/490
